lora/utils.py

```diff
@@ -112,15 +112,27 @@ def make_shards(weights: dict, max_file_size_gibibyte: int = 15):
     return shards
 
 
-def save_model(save_dir: str, weights, tokenizer, config):
+def save_model(save_dir: str, weights, tokenizer, config, de_quantize: bool = False):
     save_dir = Path(save_dir)
     save_dir.mkdir(parents=True, exist_ok=True)
-    shards = make_shards(weights)
+
+    shards = make_shards(weights, max_file_size_gibibyte=5)
```

Comment:
Curious, why did you change the max size here?

Reply:
> Curious, why did you change the max size here?

just try to it easier to upload. Otherwise, uploading 15GB takes too long and if anything happens during the upload, users need to re-upload. and, 5GB seems to be the most common size in the Huggingface repo.